docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,572 @@
+to implement and provide this capability
+
+And, finally, we have also implemented support for adding, dropping and listing partitions via
+
+- `ALTER TABLE ADD PARTITION`
+- `ALTER TABLE DROP PARTITION`
+- `SHOW PARTITIONS`
+
+**More Information**
+- [Documentation on TRUNCATE](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/truncate/)
+- [Documentation on CREATE OR REPLACE](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#create-or-replace-table)
+- [Documentation on ALTER TABLE](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/alter/#alter-table)
+- [FLIP-302: Support TRUNCATE TABLE statement in batch mode](https://cwiki.apache.org/confluence/display/FLINK/FLIP-302%3A+Support+TRUNCATE+TABLE+statement+in+batch+mode)
+- [FLIP-303: Support REPLACE TABLE AS SELECT statement](https://cwiki.apache.org/confluence/display/FLINK/FLIP-303%3A+Support+REPLACE+TABLE+AS+SELECT+statement)
+- [FLIP-305: Support atomic for CREATE TABLE AS SELECT(CTAS) statement](https://cwiki.apache.org/confluence/display/FLINK/FLIP-305%3A+Support+atomic+for+CREATE+TABLE+AS+SELECT%28CTAS%29+statement)
+
+### Time Traveling
+
+Flink supports the time travel SQL syntax for querying historical versions of data that allows users to specify a point 
+in time and retrieve the data and schema of a table as it appeared at that time. With time travel, users can easily 
+analyze and compare historical versions of data.
+
+**More information**
+- [Documentation](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/queries/time-travel/)
+- [FLIP-308: Support Time Travel](https://cwiki.apache.org/confluence/display/FLINK/FLIP-308%3A+Support+Time+Travel)
+
+## Streaming Execution Improvements
+
+### Support Operator-Level State TTL in Table API & SQL
+
+Starting from Flink 1.18, Table API and SQL users can set state time-to-live (TTL) individually for stateful operators.
+This means that for scenarios like stream regular joins, users can now set different TTLs for the left and right 
+streams. In previous versions, state expiration could only be controlled at the pipeline level using the configuration 
+`table.exec.state.ttl`. With the introduction of operator-level state retention, users can now optimize resource 
+usage according to their specific requirements.
+
+**More Information**
+- [Documentation](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/concepts/overview/#configure-operator-level-state-ttl)
+- [FLIP-292: Enhance COMPILED PLAN to support operator-level state TTL configuration](https://cwiki.apache.org/confluence/display/FLINK/FLIP-292%3A+Enhance+COMPILED+PLAN+to+support+operator-level+state+TTL+configuration)
+
+### Watermark Alignment and Idleness Detection in SQL
+
+You can now configure [watermark alignment](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/event-time/generating_watermarks/#watermark-alignment) 
```

Comment:
I think this link should be https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/event-time/generating_watermarks/#watermark-alignment-_beta_